cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -464,7 +465,7 @@ func initializeAgentAndRun(c *config.Config, logFwCfg config.LogForward) error {
 		c.FluentBitExePath,
 		c.FluentBitNRLibPath,
 		c.FluentBitParsersPath,
-		logFwCfg.FluentBitVerbose,
+		fbVerbose,
 		confTempFolder,
 	)
```

Comment:
should this be removed?

Reply:
Ah, I had not seen the `fbVerbose` being added above as a variable. After checking our [updated docs on how to set the verbose mode](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#fluentbit-logs), I believe that what you kept here is the correct way to do it. Please ignore my other rebase then.